gnomad/utils/vcf.py

```diff
@@ -960,7 +1118,9 @@ def make_vcf_filter_dict(
                 " < 20; DP < 10; and AB < 0.2 for het calls)"
             )
         },
-        "InbreedingCoeff": {"Description": f"InbreedingCoeff < {inbreeding_cutoff}"},
+        "InbreedingCoeff": {
+            "Description": f"Inbreeding coefficient < {inbreeding_cutoff}"
+        },
```

Comment:
Do we need to add `as_vqsr' too? I just saw them in the validity check.

Reply:
AS_VQSR is on L1100